### PR TITLE
Add SEAM token to Base and Base-goerli

### PR DESCRIPTION
**Pull request by fredwes**
**Description**

Adding the SEAM token along with token logo.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

@roberto-bayardo

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
How is this token to be bridged (or is it native)?   Seeing these validation failures:

> error: SEAM on chain base token 0x1C7a460413dD4e964f96D8dFC56E7223cE88CD85 could not fetch l2Bridge or l1Token.
>                     This token most likely needs nobridge or a bridge override set.
> error: SEAM on chain base-goerli token 0x8c0dE778f20e7D25E6E2AAc23d5Bee1d19Deb491 could not fetch l2Bridge or l1Token.
>                     This token most likely needs nobridge or a bridge override set.

**Comment by roberto-bayardo** [comment]
@hamdiallam should we add a no-bridge for this case? (L2 token is native but can be bridged to the L1...) or is there another way to get validation to pass?

**Comment by hamdiallam** [comment]
ahh i see, @roninjin10 had a WIP PR to allow for the reverse case to work. Is this PR close to done?

**Comment by roberto-bayardo** [comment]
Maybe we can just tag this nobridge for now and get it merged?

**Comment by roninjin10** [comment]
@roberto-bayardo I think revamping the validation might be something we doo this coming week

**Comment by roberto-bayardo** [comment]
Keeping this open, hopefully we can get this resolved after the holiday.

**Comment by hamdiallam** [comment]
cc @roninjin10 

If we dont have the validation scripts update to support this, I agree with @roberto-bayardo that we should just tag as no-bridge and merge for now

File: data/SEAM/data.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Seamless",
+    "symbol": "SEAM",
+    "decimals": 18,
+    "description": "Seamless Governance Token",
+    "website": "https://www.seamlessprotocol.com",
+    "twitter": "@SeamlessFi",
```
Comment (roberto-bayardo) [inline data/SEAM/data.json:7]: please add ` "nobridge": true,` here to get validation to pass.

**Comment by roberto-bayardo** [comment]
please add the ` "nobridge": true,` tag in your data.json for now, that should get validation to pass and we can then merge.

**Comment by roberto-bayardo** [comment]
@fredwes You might also add a comment in the data.json description that this is a native Base L2 token and "nobridge" is there purely to get validation to pass.

**Review by roberto-bayardo — APPROVED** [review]
approved for Base!

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**